### PR TITLE
feat: use buildpack

### LINKED ISSUE #6: Use buildpack

### PR DESCRIPTION
**Pull request by viceice**
closes #6

### CODE REVIEW

File: builder.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "image": "rust",
+  "startVersion": "1.33.0",
```
Comment (rarkins) [inline builder.json:3]: Any particular logic behind this? Can we start earlier and then replace it once the initial build is done?

Comment (viceice) [reply builder.json:3]: sure, it was copied from dockerfile.

**Comment by viceice** [comment]
I think i need to add some rust tests to the buildpack before merge this 🤔

**Comment by viceice** [comment]
@rarkins Strange, `build` status no longer match since matrix job :-(

**Comment by rarkins** [comment]
fixed

**Pull request merged**